indico/core/storage/backend_test.py

```diff
@@ -102,15 +102,21 @@ def test_fs_dir(fs_storage):
 
 
 def test_fs_operations(fs_storage):
-    f1 = fs_storage.save('foo/bar/test.txt', 'unused/unused', 'unused', b'hello world')
-    f2 = fs_storage.save('foo/bar/test2.txt', 'unused/unused', 'unused', b'hello there')
-    f3 = fs_storage.save('test.txt', 'unused/unused', 'unused', b'hello test')
+    f1, h1 = fs_storage.save('foo/bar/test.txt', 'unused/unused', 'unused', b'hello world')
+    f2, h2 = fs_storage.save('foo/bar/test2.txt', 'unused/unused', 'unused', b'hello there')
+    f3, h3 = fs_storage.save('test.txt', 'unused/unused', 'unused', b'very very long file' * 1024 * 1024)
+
+    # check md5 checksums
+    assert h1 == b'5eb63bbbe01eeed093cb22bb8f5acdc3'
```

Comment:
I would rather return the hashes as unicode. They're also unicode when stored in the DB.